bundle/android/strings.xml

```diff
@@ -297,6 +297,7 @@ translators: %s: Select control button label e.g. "Button width" -->
     <string name="gutenberg_native_replace_image_or_video" tools:ignore="UnusedResources">Replace image or video</string>
     <string name="gutenberg_native_replace_video" tools:ignore="UnusedResources">Replace video</string>
     <string name="gutenberg_native_retry" tools:ignore="UnusedResources">Retry</string>
+    <string name="gutenberg_native_reusable" tools:ignore="UnusedResources">Reusable</string>
```

Comment:
This addition results from merging https://github.com/WordPress/gutenberg/pull/51144, but not yet https://github.com/WordPress/gutenberg/pull/51704.